### ISSUE TITLE
Cannot minimize webms by clicking on them.

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#3221]
Submitter: Wang
- Firefox 94.0.1 (64-bit)
- Tampermonkey v4.13.6136
- 4chan X 1.14.21.7

After left clicking on a webm to expand it, I cannot minimize it by left clicking on it a second time, instead it just pauses the webm. Subsequent left clicks alternate play and pause. I discovered that if I disable the “Show Controls” option in the 4chan X settings under Images and Videos, I am then able to minimize webms by left click, however this leaves me without controls.

Note: In Chrome, with Tampermonkey v4.13 and 4chan X 1.14.21.7, I can minimize an expanded webm by left clicking it without disabling the show controls option.

**Comment by kakhulu31** [comment]
Yes, minimizing doesn't work on firefox if html player controls are enabled.
